src/sdk/analytics/hooks/useGetClientId.ts

```diff
@@ -0,0 +1,40 @@
+import { useRef } from 'react'
+
+const GA_CLIENT_ID_KEY = 'main::store::ga::client_id'
+
+// https://gist.github.com/kindy/35ccd6c60b650e7fd261
```

Comment:
Please add a description here. I know this symbolizes where this code comes from, but only because I've seen it before. Adding a description makes it so clear that I don't even have to visit the link to understand what it means.